tabs/hide-tabs-macos.css

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /*
- * This style will hide the tab bar. For MacOS
+ * This style will hide the tab bar. For MacOS Big Sur
```

Comment:
Do you expect this to break on older versions of MacOS?

Reply:
Yeah, the spacing is designed for Big Sur's redesign, since #180, it just wasn't documented.

Reply:
Oh, okay... Oh well, nobody has complained since then, so let's not block on it :)